src/index.jsx

```diff
@@ -36,6 +37,7 @@ function createInitialState() {
 function configure() {
   const { store, history } = configureStore(createInitialState(), configuration.ENVIRONMENT);
 
+  configureOrderHistoryServices({ configuration, apiClient });
   configureUserAccountApiService(configuration, apiClient);
```

Comment:
Mentioned below as well, but I'd probably just make the `configureOrderHistoryServices` params two separate params, like `configureUserAccountApiService` here.  passing them in inside an object doesn't get us much, as we just unpack them on the other side.

(Also, naming: `configureOrderHistoryApiService` - 'ApiService" for consistency...